### PR TITLE
Joel cp4aiops issue 205

### PR DESCRIPTION
**Pull request by jgod1360**

### CODE REVIEW

**Review by annumberhocker — CHANGES_REQUESTED** [review]
Please address comments, then let me know and I will approve and merge.

**Comment by annumberhocker** [comment]
Please resolve all merge conficts

File: examples/cp4aiops/variables.tf [hunk 1]
```diff
@@ -27,6 +31,11 @@ variable "entitled_registry_user_email" {
   description = "Docker email address"
 }
 
+variable "cp4aiops_namespace" {
+  default = "cp4aiops"
+  description = "Namespace for Cloud Pak for AIOps"
+}
+
 locals {
   cluster_config_path = "./.kube/config"
```
[the remark this answers is not in the document]
Comment (jgod1360) [reply examples/cp4aiops/variables.tf:40]: I updated it to: 
```
variable "cluster_config_path" {
  description = "Path to the Kubernetes configuration file to access your cluster"
  default     = "./.kube/config"
}```

**Review by jgod1360 — COMMENTED** [review]
I fixed the cluster config path variable
```
variable "cluster_config_path" {
  description = "Path to the Kubernetes configuration file to access your cluster"
  default     = "./.kube/config"
}
```

File: modules/cp4aiops/variables.tf [hunk 2]
```diff
@@ -22,7 +18,7 @@ variable "cluster_config_path" {
 }
 
 variable "openshift_version" {
-  default     = "4.7"
+  default     = "4.6"
```
Comment (annumberhocker) [inline modules/cp4aiops/variables.tf:21]: Looks like this got backleveled to 4.6.   Needs to be 4.7

**Pull request closed without merging**